tests/typing_test_data.py

```diff
@@ -35,14 +35,18 @@ def f5(x: int, *, y: str, z: str) -> None:
     pass
 
 
-def f6(x: int = None, y: dict = {}) -> None:
```

Comment:
Is there a particular reason you're inserting a function here, instead of appending it to the list?